docs/cluster-management/uninstall.md

```diff
@@ -43,6 +43,4 @@ aws s3 rb --force s3://<bucket>
 aws logs describe-log-groups --log-group-name-prefix=<log_group> --query logGroups[*].[logGroupName] --output text | xargs -I {} aws logs delete-log-group --log-group-name {}
 ```
 
-If you've setup API gateway and want to delete it, please follow these [instructions](../guides/api-gateway.md#cleanup).
-
-If you've configured HTTPS by specifying an SSL Certificate for a subdomain in your cluster configuration, you may wish to remove the SSL Certificate and Hosted Zone for the domain by following these [instructions](../guides/subdomain-https-setup.md#cleanup).
+If you've configured a custom domain for your APIs, you may wish to remove the SSL Certificate and Hosted Zone for the domain by following these [instructions](../guides/custom-domain.md#cleanup).
```

Comment:
I think we can simplify this by making a guide on how to add custom [domains](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-custom-domain-names.html) with API gateway which is easier!